hever/inc/wpcom-editor-colors.php

```diff
@@ -1,2 +1,2 @@
 <?php
-require_once get_template_directory() . '/inc/wpcom-colors-css-variables.php';
\ No newline at end of file
+require_once get_template_directory() . '/inc/wpcom-colors-css-variables.php';
```

Comment:
I don't understand this one from the rebase

Reply:
Files should have an empty blank line at the end.

Reply:
mmh, the file that has been commited has the blank line and the one that comes from the rebase doesn't, I think that was my phpcs fixing that for me